tests/integration/repo_fork_test.go

```diff
@@ -74,3 +79,64 @@ func TestRepoForkToOrg(t *testing.T) {
 	_, exists := htmlDoc.doc.Find(`a.ui.button[href*="/fork"]`).Attr("href")
 	assert.False(t, exists, "Forking should not be allowed anymore")
 }
+
+func TestForkListLimitedAndPrivateRepos(t *testing.T) {
+	forkItemSelector := ".tw-flex.tw-items-center.tw-py-2"
```

Comment:
It can't be a right selector

Reply:
<img width="484" alt="image" src="https://github.com/user-attachments/assets/e980e893-8fd8-49e2-9c45-6c9b502f62f5">
I found the fork repository item have classes like this, do you have any better idea?

Reply:
`a[href~="/repo-name"]`

Reply:
How does that work? We don't know which repositories will be listed.

Reply:
It is in a unit test, why can't we know ......